test/controllers/returns/ImportedPlasticPackagingControllerSpec.scala

```diff
@@ -181,19 +189,31 @@ class ImportedPlasticPackagingControllerSpec extends SpecBase with MockitoSugar
 
     "submit must redirect to the mini-cya page if the answer has not changed" in {
 
-      when(mockUserAnswers.change(any(), any())(any())).thenReturn(None) // Respond saying the user-answer hasn't changed
-      when(mockUserAnswers.get(eqq(ImportedPlasticPackagingPage))(any())).thenReturn(Some(true)) // TODO shouldn't need this?
+      // Check change_v3, saveUserAnswerFunc, manufacturedPlasticPackagingRoute, returned redirect val
+      
+      // user answers responds with "answer not changed"
```

Comment:
As above.

Reply:
Checks done and that comment removed. I've kept (and slightly changed) the comment about "answer not changed" as it explains why that when() clause is there and what it does. If we overhauled the tests, we could make this test much shorter then arguably it would be clearer why that when() is there.